clang/include/clang/AST/Expr.h

```diff
@@ -1914,6 +1914,17 @@ class StringLiteral final
     llvm_unreachable("Unsupported character width!");
   }
 
+  // Get code unit but preserve sign info.
+  int64_t getCodeUnitS(size_t I, uint64_t ByteWidth) const {
```

Comment:
```suggestion
  int64_t getCodeUnitS(size_t I, uint64_t BitWidth) const {
```
ByteWidth is actually a number of bits, right?

Reply:
It is a number of bits in byte, but ok